### PR TITLE
Add a programm that recognize  Fibonacci number.

### PR DESCRIPTION
**Pull request by YuliaMiranovich**

### CODE REVIEW

File: DEV-3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                if (number_to_check < 0)
+                 {
+                    Console.Write("It's a negative number. Try again: ");
+                    Main();
+                 }
+                else
+                {
+                    bool result = false;
+                    Fibonacci_numbers number = new Fibonacci_numbers();
+                    result = number.Check_for_Fibonacci_numbers(number_to_check);
```
Comment (IgorKheidorov) [inline DEV-3/Program.cs]: Code style, make name not c-like, better name is numbersFibinacci.

File: DEV-3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                int number_to_check = int.Parse(Console.ReadLine());
+                if (number_to_check < 0)
+                 {
+                    Console.Write("It's a negative number. Try again: ");
+                    Main();
+                 }
+                else
+                {
+                    bool result = false;
+                    Fibonacci_numbers number = new Fibonacci_numbers();
```
Comment (IgorKheidorov) [inline DEV-3/Program.cs]: Class name is to be FibonacciNumber (code style)

File: DEV-3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+        }
+       internal bool Check_for_Fibonacci_numbers(int number_to_check)
+        {
+            int previos1 = 0;
+            int previos2 = 0;
+            int subsequent = 1;
+            bool verification_for_Fibonacci = false;
+
+            for (int i = 0; i < number_to_check; i++)
```
Comment (IgorKheidorov) [inline DEV-3/Program.cs]: Number to check has to be included (<=)

Comment (YuliaMiranovich) [reply DEV-3/Program.cs]: But if the numberToCheck is a Fibonacci number, then the loop will check this maximum of numberToCheck-1 iteration (which start at 0). But maybe I'm wrong ...

File: DEV-3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        static void Main()
+        {
+            try
+            {
+                Console.WriteLine("Enter a number: ");
+                int numberToCheck = int.Parse(Console.ReadLine());
+                if (numberToCheck < 0)
+                {
+                    Console.Write("It's a negative number. Try again: ");
+                    Main();
```
Comment (IgorKheidorov) [inline DEV-3/Program.cs:17]: Recursion is not a good idea, use other decision

File: DEV-3/Program.cs [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                }
+                else
+                {
+                    bool result = false;
+                    FibonacciNumber numbersFibonacci = new FibonacciNumber();
```
Comment (IgorKheidorov) [inline DEV-3/Program.cs:22]: no need to create object of FibonacciNumber  here, you already in this object

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**